apisix/init.lua

```diff
@@ -110,6 +110,11 @@ function _M.http_init_worker()
     end
     require("apisix.balancer").init_worker()
     load_balancer = require("apisix.balancer")
+
+    if core.config == require("apisix.core.config_shdict") then
```

Comment:
I think we may need to unify the term, the shdict is the container, the xds is the way, I think here `config_xds` will be better.

Reply:
@membphis I don't think so, from the point of view of Apache APISIX, what is `amesh`? Why do we need to care about a commercial product?